server/controllers/request.js

```diff
@@ -0,0 +1,95 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+const {
+  updateRequest,
+  checkQueryForEmptyFields,
+  validateQueryForNewRequest,
+} = require("../utils/helperFunctions");
+
+// @route GET /request/load
+// @desc Get list of requests for logged-in user
+// @access Private
+exports.loadRequests = asyncHandler(async (req, res, next) => {
+  const uid = req.user.id;
+  let requests;
+
+  if (uid) {
+    requests = await Request.find({
+      userId: uid,
+    });
+  }
+
+  res.status(200).json(requests);
+});
+
+// @route POST /request/new
+// @desc Create a new request
+// @access Public
```

Comment:
Access needs to be private instead, we only want authenticated users to make requests.